dataAccessObjects.py

```diff
@@ -244,6 +244,15 @@ def generateNewId(self):
             return "imge{0:010d}".format(int(res[0][0][4:]) + 1)
         else:
             return "imge0000000001"
+    def getImgCheck(self,imgId):
```

Comment:
Please change the function name, a good function name should describe its behavior.
We can probably know what this function will do when take a glance at the function name